watcher_events.go

```diff
@@ -78,12 +78,15 @@ func buildSentryEventFromGeneralEvent(ctx context.Context, event *v1.Event, scop
 
 	involvedObj, ok := findObject(ctx, event.InvolvedObject.Kind, event.InvolvedObject.Namespace, event.InvolvedObject.Name)
 
-	// cannot find event
+	// Cannot find the involved object related to the event
+	// note: this may mean the object is not a supported kind
+	// (e.g. Node, Service, StatefulSets)
 	if !ok {
+		runCommonEnhancer(ctx, scope, sentryEvent)
```

Comment:
Looks like we're not running `eventEnhancer` right now?
Can we still run `runEnhancers`, and just pass `nil` for object (with proper handling inside)?

Reply:
I refactored `runEnhancers` so the object can just be `nil`. The function will only call `eventEnhancer` and `objectEnhancer` if their corresponding object is not `nil`.